app-frontend/src/app/pages/projects/edit/annotate/annotate.scss

```diff
@@ -0,0 +1,97 @@
+@import "../../../../../assets/styles/sass/settings/_colors.scss";
+.sidebar-project.no-annotation,
```

Comment:
@designmatty just noting that the annotation styles are here, instead of in _shame.scss